src/module.js

```diff
@@ -1434,11 +1462,18 @@ export default class DailyIframe extends EventEmitter {
             p.screenVideoTrack = screenVideoTracks[0].pendingTrack;
           } else if (!screenVideoTracks[0].pendingTrack.muted) {
             // otherwise, add the found track if it's not muted
+            // note: there is an issue here with timing ... Chrome (and
+            // possibly other browsers), gets a video track that's initially
+            // not muted, for an audio-only screenshare. The track
+            // switches to muted fairly quickly, but we don't have any
+            // logic in place to respond to that. todo: fix this so that,
+            // at the very least we get a track-stopped event when the
+            // "empty" track switches to muted.
```

Comment:
Why doesn't this work for remote participants at least? We add the an `onmute` handler as soon as we get a remote track from a participant...